pyVHDLParser/CLI/__init__.py

```diff
@@ -25,8 +25,9 @@
 from pyAttributes                     import Attribute
 from pyAttributes.ArgParseAttributes  import ArgumentAttribute, SwitchArgumentAttribute
 
-from pyVHDLParser.Token               import LinebreakToken, IndentationToken, CommentToken, StringLiteralToken
-from pyVHDLParser.Token               import IntegerLiteralToken, WordToken, Token, SpaceToken, CharacterToken
+from pyVHDLParser.Token               import (LinebreakToken, IndentationToken, CommentToken, StringLiteralToken,
+											  IntegerLiteralToken, WordToken, SpaceToken, CharacterToken)
+from pyVHDLParser.Token               import Token as BaseToken
```

Comment:
@Paebbels, since the motivation is a name duplication/conflict, maybe it's sensible to change Token -> BaseToken in the codebase and not just here?